cmd/schema-tweak/main.go

```diff
@@ -0,0 +1,232 @@
+/*
+Copyright 2024 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"gopkg.in/yaml.v3"
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+func main() {
+	resourcesDir := "config/core/300-resources"
+	files, err := os.ReadDir(resourcesDir)
+	if err != nil {
+		log.Fatalln("failed to read CRD directory", err)
+	}
+
+	for _, file := range files {
```

Comment:
Although it is fast already could this be done concurrently? :thinking:

Reply:
I don't think it's worth the complexity this would introduce - eg. right now I just `log.Fatalf` which quits the program.

Doing it concurrently would require changing it to return errors etc.